mne_bids/tests/test_read.py

```diff
@@ -456,6 +456,14 @@ def test_handle_info_reading(tmpdir):
     with pytest.raises(ValueError, match="PowerLineFrequency .* required"):
         write_raw_bids(raw, bids_path, overwrite=True)
 
+    # check whether there are "Extra points" in raw.info['dig'] if
+    # DigitizedHeadPoints is set to True and not otherwise
```

Comment:
I assume that for this check you manually found that the dataset being read here does indeed have `DigitizedHeadPoints=True` in its JSON file? Would be nicer to make this explicit in the code - but I'd also accept a slightly more verbose comment describing that.

also, the "... and not otherwise" case is not really tested here, is it?

Reply:
True, I made this one more explicit now.